assets/js/phoenix_live_view.js

```diff
@@ -1549,6 +1549,14 @@ export let DOM = {
     }
   },
 
+  showError(inputEl, phxFeedbackFor){
+    if (inputEl.id) {
```

Comment:
```suggestion
    if(inputEl.id){
```

Reply:
@chrismccord I notice I'm having the wrong spacing a lot. Is there a js formatter/linter you use? When I use whatever autoformat vscode comes with it changes spacing/indents in the entire file. What should I be using?

Reply:
I don't use a formatter/linter. No worries :)